openbb_terminal/cryptocurrency/defi/defi_controller.py

```diff
@@ -440,11 +440,9 @@ def call_gdapps(self, other_args: List[str]):
             default=40,
         )
 
-        ns_parser = self.parse_known_args_and_warn(
+        if ns_parser := self.parse_known_args_and_warn(
```

Comment:
I am not sure if I'll be changing all of them. Any thoughts regarding if we should end up changing all of them or not?

Reply:
`There should be one-- and preferably only one --obvious way to do it.` <- The Zen Of Python =)

Reply:
Then that's your guys' call. I'll try and change all controllers to be like that. But, A few of them MIGHT slip trough. Sometimes my fingers are a little bit too fast on nvim 😅

Reply:
Now that I think about it. Perhaps we can go as far as not even assigning a value. In many of the cases, it seems like we just checks of an output is given, but we do not use value at all. This does not apply to this spesific case, but or the changes I introduced in openbb_terminal/mutual_funds/mutual_fund_controller.py, line number 485.
That can be shortened to just:
```python
if self.parse_known_args_and_warn(parser, other_args):
** rest of if flow here
```
Again, this would be a tradeoff between readability and reduction of variable assignments.

Reply:
And there, I would argue. ns_parser does not(at least to me), explain what the variable does(not that it is being used)

Reply:
> Then that's your guys' call. I'll try and change all controllers to be like that. But, A few of them MIGHT slip trough. Sometimes my fingers are a little bit too fast on nvim 😅

A fellow VIM user lets go!!!!!!!!!!!!!!!!!!!